sections/pyenv.zsh

```diff
@@ -23,7 +23,7 @@ spaceship_pyenv() {
   [[ $SPACESHIP_PYENV_SHOW == false ]] && return
 
   # Show pyenv python version only for Python-specific folders
-  [[ -f requirements.txt ]] || [[ -n *.py(#qN^/) ]] || return
+  [[ -f requirements.txt ]] || [[ -n *.py(#qN^/) ]] || [[ -f pyproject.toml ]] || return
```

Comment:
```suggestion
  [[ -f requirements.txt  || -f pyproject.toml || -n *.py(#qN^/) ]] || return
```

Reply:
Yeah, you're right, thats better. I just added it to the PR